### ISSUE TITLE
Unreadable table

### ISSUE THREAD

**Issue by megagrump** [hirschan/Dark-Mode-Wikipedia#21]
![image](https://user-images.githubusercontent.com/31128870/170778345-9fca8849-9c64-4568-b3af-76a14f7533c6.png)

https://en.wikipedia.org/wiki/Amateur_radio_call_signs#Formation_of_an_amateur_radio_call_sign

**Comment by hirschan** [comment]
Thanks, will fix immediately

**Comment by megagrump** [comment]
Also here: https://en.wikipedia.org/wiki/Piano_key_frequencies

![image](https://user-images.githubusercontent.com/31128870/170814900-fe7879da-8d8c-4031-9df5-919e912cbe81.png)

**Comment by hirschan** [comment]
Good catch - I have fixed now and will upload later, please let me know if you find more tables. 

The issue with tables is that they have `td` elements and I cannot override all `td `elements to dark because that would mess up some other tables, for example this one here would be completely dark: [https://es.wikipedia.org/wiki/King_Crimson#Miembros_pasados](https://es.wikipedia.org/wiki/King_Crimson#Miembros_pasados) and also temperature tables with colors would be replaced with dark.

**Comment by megagrump** [comment]
Sure, I'll gladly complain about anything that isn't as perfect as it should be! lol

Thanks for the effort and I commend your taste in music too if you like King Crimson.

**Comment by hirschan** [comment]
It should be fixed now, included in the latest update. Haha actually I don't listen to King Crimson - I just remember some user linking me it when I messed up the table colors ;)

**Comment by HanatakeYurii** [comment]
I have found something similar, not exactly tables but same problem:
[https://en.wikipedia.org/wiki/Emperor_Monmu]
![firefox_llrPQ9thhM](https://user-images.githubusercontent.com/5856491/176439360-1c461747-7fbe-43c0-b826-d9e13f388859.png)
[https://en.wikipedia.org/wiki/Mineral_(nutrient)]
![firefox_AaXtGCwdW4](https://user-images.githubusercontent.com/5856491/176439364-108e7463-b7d2-403e-8a58-4b6db29f2742.png)

**Comment by HanatakeYurii** [comment]
Another occurrence in another particular case implying special characters (phonological):
<https://en.wikipedia.org/wiki/Affricate>
![firefox_47FaKQR5Hd](https://user-images.githubusercontent.com/5856491/176864501-0c725ca2-b532-4fc0-bee2-801e55e7a635.png)
As you can (not) see some characters are black on black thus not showing.

**Comment by hirschan** [comment]
Yeah I noticed that too, seems like some changes became overridden. Will look into this soon!

**Comment by HanatakeYurii** [comment]
Thanks

**Comment by hirschan** [comment]
List of other tables I found (adding them here to keep an eye of them)
https://en.wikipedia.org/wiki/The_Economist_Democracy_Index#By_region
https://en.wikipedia.org/wiki/2024_United_States_presidential_election#Forecasts
https://en.wikipedia.org/wiki/London#Climate

**Comment by Korb** [comment]
Before using the extension:
![2024-07-10_20-32-59](https://github.com/hirschan/Dark-Mode-Wikipedia/assets/3509797/4ea9b804-ac65-4b1a-8482-89cf19f0c86e)
After using the extension:
![2024-07-10_20-32-35](https://github.com/hirschan/Dark-Mode-Wikipedia/assets/3509797/68059645-1725-439a-9bde-dd845809ba08)
https://en.wikipedia.org/wiki/XCOM#Released

Dark Mode for Wikipedia 4.5.7 (July 7, 2024)
Mozilla Firefox 129.0b1 (64-bit)
